lib/javascript/JavascriptModulesPlugin.js

```diff
@@ -710,6 +717,24 @@ class JavascriptModulesPlugin {
 			buf.push("");
 		}
 
+		if (runtimeRequirements.has(RuntimeGlobals.hasOwnProperty)) {
```

Comment:
This should not be here, but a separate HelperRuntimeModule referenced in `RuntimePlugin.js`